NetCord/Application.cs

```diff
@@ -1,41 +1,20 @@
-﻿using NetCord.Rest;
+using NetCord.Rest;
 
 namespace NetCord;
 
+/// <summary>
+/// Applications or 'apps', are containers for developer platform features, and can be installed to guilds and/or user accounts.
```

Comment:
Don't apps contain bots that were later renamed to apps? Like I think the Application object is purely the object that contains the bot. There is even a `Bot` property